src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static void main(String[] args) throws DukeException {
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+         */
+        Scanner sc = new Scanner(System.in);
+        String Hello = "Hello! I'm Duke\nWhat can i do for you?";
+        System.out.println(Hello);
+
+        int flag = 0;
+        ArrayList<task> mylist = new ArrayList<>();
+
+        while(flag == 0) {
+            String input = sc.nextLine();
+            String[] inarr = input.split(" ");
+            String Line = "_________________________________________________________________";
+
+            System.out.println(Line);
+            try {
+                if (input.equals("bye")) {
```

Comment:
To add on to Jeremy's word. I believe it's generally a good practice to standardize your code, so from using `input.equals` to suddenly switching to `inarr[0]` may lead to confusion as to why you did that. Additionally `inarr` should be named as `inArr` instead